examples/nextjs-hooks/pages/about.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import '../src/bootstrap';
```

Comment:
This shouldn't be necessary from my understanding since we import this already in `_app.js` and `_app.js` is executed on client and server.

Reply:
Console log the execution order between the client and the server, it's different. We need to call it first. At least, until v4 is out.